delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/ReferenceExchange.scala

```diff
@@ -0,0 +1,68 @@
+package ch.epfl.bluebrain.nexus.delta.sdk
+
+import ch.epfl.bluebrain.nexus.delta.rdf.RdfError
+import ch.epfl.bluebrain.nexus.delta.rdf.graph.{Dot, NTriples}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.{CompactedJsonLd, ExpandedJsonLd}
+import ch.epfl.bluebrain.nexus.delta.sdk.ReferenceExchange.ReferenceExchangeValue
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{ResourceF, ResourceRef}
+import io.circe.Json
+import monix.bio.{IO, UIO}
+
+/**
+  * Contract definition for registering and consuming the ability to retrieve resources in a common JSON-LD format.
+  * Plugins can provide implementations for custom resource types such that those resources can be handled in an
+  * uniform way. Examples of use-cases would be: the ability to register resources defined by plugins (e.g. views) to
+  * be included in archives or being able to implement a generic `GET /resources/{org}/{proj}/_/{id}` endpoint.
+  */
+trait ReferenceExchange {
```

Comment:
This is still wrong. You haven't merge this code with `EventExchange` as @imsdu and me pointed out

Reply:
This will be handled later.